### PR TITLE
ICS20: Simplify sendPacket interface

### PR DESCRIPTION
**Pull request by mpoke**
Makes ICS20 consistent with https://github.com/cosmos/ibc/pull/731

Closes https://github.com/cosmos/ibc/issues/709

### CODE REVIEW

File: spec/app/ics-020-fungible-token-transfer/README.md [hunk 1]
```diff
@@ -203,35 +203,43 @@ In plain English, between chains `A` and `B`:
...
 
-`createOutgoingPacket` must be called by a transaction handler in the module which performs appropriate signature checks, specific to the account owner on the host state machine.
+`sendFungibleTokens` must be called by a transaction handler in the module which performs appropriate signature checks, specific to the account owner on the host state machine.
 
 ```typescript
-function createOutgoingPacket(
+function sendFungibleTokens(
   denomination: string,
   amount: uint256,
-  sender: string,
```
Comment (AdityaSripal) [inline spec/app/ics-020-fungible-token-transfer/README.md:212]: We still need `sender`

Comment (mpoke) [reply spec/app/ics-020-fungible-token-transfer/README.md:212]: Yeah, I wanted to remove source.

**Review by AdityaSripal — APPROVED** [review]
